Country.py

```diff
@@ -208,12 +226,23 @@ def propagation(self, other):
     """Modify Time + XXXX to modify the speed of the game."""
     if pygame.time.get_ticks() > (Time + 1000):
         Time = pygame.time.get_ticks()
+        """ If population == 0 then game is over"""
         if all(country.max_pop == 0 for country in countries) == True:
             running = False
             endscreen = True
         #print ('For each country: (infected ratio, total population)', (country1.infected_ratio(),country1.max_pop), (country2.infected_ratio(),country2.max_pop), (country3.infected_ratio(),country3.max_pop))
```

Comment:
I suggest removing print statements and logs before submitting pull requests (in the future)